File: Server/Joueur.ts
export class Joueur {
  prenom: string;
  nom: string;
  age: number;
  rang: number;
  pays: string;
  constructor(prenom, nom, age, rang, pays) {
    this.prenom = prenom;
    this.nom = nom;
    this.age = age;
    this.rang = rang;
    this.pays = pays;
  }
}

File: TP1-Android/app/src/main/java/com/example/tp1_b/Models/Joueur.kt
package com.example.tp1_b.Models

import kotlinx.serialization.Serializable

@Serializable
class Joueur(
    val prenom: String = "<no first name>",
    val nom: String = "<no last name>",
    val age: Int = -1,
    val rang: Int = -1,
    val pays: String = "<no country>",
)